src/settings.rs

```diff
@@ -0,0 +1,73 @@
+//use crate::LOG_DRAIN;
+
+use serde::{Deserialize, Serialize};
+//use slog::info;
+
+// Describe the settings your policy expects when
+// loaded by the policy server.
+#[derive(Serialize, Deserialize, Default, Debug)]
+#[serde(default)]
+#[serde(rename_all = "camelCase")]
+pub(crate) struct Settings {
+    pub reserved_runtimes: Vec<String>,
+    pub fallback_runtime: Option<String>,
+    pub default_runtime_reserved: Option<bool>,
+}
+
+impl kubewarden::settings::Validatable for Settings {
+    fn validate(&self) -> Result<(), String> {
+        if let Some(fallback_runtime) = &self.fallback_runtime {
+            if self.reserved_runtimes.contains(fallback_runtime) {
+                return Err(format!(
+                    "fallback runtime {} cannot be part of the reserved runtimes",
+                    fallback_runtime
+                ));
+            }
+        }
+        Ok(())
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use rstest::rstest;
+    use serde_json::json;
+
+    use crate::kubewarden::settings::Validatable;
+
+    #[test]
+    fn json_serialization_test() -> Result<(), serde_json::Error> {
+        // Ensure the rust version of the policy serialize settings as the swift one
+        let settings = Settings {
+            reserved_runtimes: vec!["runc".to_string()],
+            fallback_runtime: Some("gvisor".to_string()),
+            default_runtime_reserved: Some(true),
+        };
+        let expected_json = json!({
+            "reservedRuntimes": ["runc"],
+            "fallbackRuntime": "gvisor",
+            "defaultRuntimeReserved": true,
+        });
+        let json_value = serde_json::to_value(settings)?;
+        assert_eq!(json_value, expected_json);
+        Ok(())
+    }
```

Comment:
I think we can remove this test. I appreciate the concerns about breaking the settings between the Rust and the Swift version, but this is a one time thing (the swift -> rust migration)

Reply:
Yes, I've added this during development and forgot to remove it. Thanks for the heads up